qiskit_sphinx_theme/furo/base/static/styles/qiskit_changes.css

```diff
@@ -367,10 +367,75 @@ html {
   text-transform: uppercase;
 }
 
+/* ---------------------------------------------------------------------------
+* Drop shadows
+* ---------------------------------------------------------------------------- */
+
+.admonition,
+.topic,
+table.docutils,
+kbd:not(.compound) {
+  border-radius: 0;
+  box-shadow: none;
+}
+
+/* Restore borders for tables by overriding rules that disabled them. */
+table.docutils td:last-child,
+table.docutils th:last-child {
+  border-right: 1px solid var(--color-table-border);
+}
+table.docutils td:first-child,
+table.docutils th:first-child {
+  border-left: 1px solid var(--color-table-border);
+}
+
+/* We have a specificity war with Sphinx design elements. The `:not(#specifity-war)` selector
+*  works around that. */
+:not(#specifity-war).sd-card,
+:not(#specifity-war).sd-tab-content,
+:not(#specifity-war).sd-shadow-sm,
+:not(#specifity-war).sd-shadow-md,
+:not(#specifity-war).sd-shadow-lg {
+  box-shadow: none !important;
```

Comment:
eeek when I see `!important` in css thats a red flag. Is there no way to do this without `!important`? Possibly by unsetting the box shadow at a higher level?

Reply:
ha! ok fair enough 😅